vnets.tf

```diff
@@ -82,6 +80,13 @@ resource "azurerm_virtual_network" "trusted" {
   address_space       = ["10.252.0.0/21"] # 10.252.0.1 - 10.252.7.254
   tags                = local.default_tags
 }
+resource "azurerm_virtual_network" "public_db" {
```

Comment:
I would have put this vnet definition next to the others in https://github.com/jenkins-infra/azure-net/blob/main/vnets.tf for an easier view of the different vnet ranges.